cfgov/scripts/http_smoke_test.py

```diff
@@ -78,6 +86,7 @@
     '/data-research/cfpb-research-conference/',
     '/data-research/consumer-complaints/',
     '/data-research/hmda/',
+    '/data-research/hmda/for-filers', # This URL 404s if it has a trailing slash
```

Comment:
`flake8` will complain if you don't have two spaces before a comment. See [PEP8](https://www.python.org/dev/peps/pep-0008/#inline-comments):

>  Inline comments should be separated by at least two spaces from the statement.